environment.yml

```diff
@@ -1,10 +1,12 @@
-name: calicost_env
+name: calicost
```

Comment:
This corresponds to condo environment name, right? If so, shall we change it back as the README and readthedocs documentations still have `conda activate calicost_env` in the installation instruction.